app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageRecsFragment.kt

```diff
@@ -83,6 +84,8 @@ class SuggestedEditsImageRecsFragment : SuggestedEditsItemFragment(), MenuProvid
                 .show()
             ImageRecommendationsEvent.logAction("editsummary_success_confirm", "editsummary_dialog",
                 getActionStringForAnalytics(acceptanceState = "accepted", revisionId = revId, addTimeSpent = true), viewModel.langCode)
+            val title = PageTitle(viewModel.recommendation.titleText, WikiSite.forLanguageCode(viewModel.langCode))
```

Comment:
Can we use `viewModel.pageTitle` instead of creating a new `PageTitle` here?

Please help to update the same thing in `requestEdit` launcher, thanks!